src/js/content/content.js

```diff
@@ -21,7 +21,7 @@ window.onload = () => {
   chrome.runtime.sendMessage({ fragmentsData: findFragmentsInContent() }, (response) => {
     /* eslint-disable no-console */
     if (response.status === status.error) {
-      console.warn(response.msg);
+      console.log(response.msg);
```

Comment:
What level is valid here?

Reply:
Since this error occurs for pages that just don't use (debug) knotx, do we even want to log it?

Reply:
fixed iin https://github.com/Knotx/knotx-fragments-chrome-extension/pull/30/commits/b9d77e217c7cd7e95e466970616696f62b3943d0